x-pack/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -17,88 +17,195 @@ import {
   ALERTS_HEADERS_TARGET_IMPORT_HASH,
   ALERTS_HEADERS_RULE_DESCRIPTION,
 } from '../../../detections/components/alerts_table/translations';
-import {
-  AGENT_STATUS_FIELD_NAME,
-  IP_FIELD_TYPE,
-} from '../../../timelines/components/timeline/body/renderers/constants';
-import { DESTINATION_IP_FIELD_NAME, SOURCE_IP_FIELD_NAME } from '../../../network/components/ip';
+import { AGENT_STATUS_FIELD_NAME } from '../../../timelines/components/timeline/body/renderers/constants';
 import { getEnrichedFieldInfo, SummaryRow } from './helpers';
 import { EventSummaryField } from './types';
 import { TimelineEventsDetailsItem } from '../../../../common/search_strategy/timeline';
 
 import { isAlertFromEndpointEvent } from '../../utils/endpoint_alert_check';
-import { EventCode } from '../../../../common/ecs/event';
+import { EventCode, EventCategory } from '../../../../common/ecs/event';
 
+/** Always show these fields */
 const defaultDisplayFields: EventSummaryField[] = [
```

Comment:
Thanks for the comment here! Think renaming this to `alwaysDisplayedFields` would be helpful?

Reply:
Sounds good to me! -> https://github.com/elastic/kibana/pull/123239/commits/7da08fb03071f4bab08b9ec607e3a8e08b01d4eb